_posts/2021-08-13-multi-factor-authentication-with-twilio-verify-api.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Multi Factor Authentication: A Developer’s Perspective on Working with Twilio Verify API"
+slug: "multi-factor-authentication-dev-perspective-on-working-with-twilio-verify-api"
+category: blog
+post_image: /assets/images/multi-factor-authentication/header_image.png
+author: seosamh
+date: 2021-08-13 09:30:00
+excerpt: "Multi Factor Authentication is a method of authentication that adds extra steps of verification for gaining access to resources such as an online account, internal system, application or network. Authentication has been traditionally completed by validating a username and password."
+graphic: /assets/images/multi-factor-authentication/header_image.png
+---
+
+### What is Multi Factor Authentication (MFA)?
+
+Multi Factor Authentication is a method of authentication that adds extra steps of verification for gaining access to resources such as an online account, internal system, application or network. Authentication has been traditionally completed by validating a username and password. However, there is now a need for higher security as cybercrime increases and evolves. Techniques such as brute force attacks, and social engineering have weakened conventional security measures for restriction of access, and they no longer provide sufficient guard against vulnerability. There are also regulations such as PSD2 in the European Union that place an onus on certain industries to ensure enforcement of strong customer authentication.
+
+As a Software Engineer at OnePageCRM, I have recently worked on the implementation of Two Step Authentication (2SA), which is a type of Multi Factor Authentication. The first step in completing this project was to consider which factors provide the greatest security for our users during the login process.
```

Comment:
two step isn't a type of multi factor. Our two step is a single factor (knowledge). 

Also "greatest security" isn't the only thing we considered. Also ease of use is just as important. Maybe mentions ease of use too,